linux/sysvinit/ckb-next-daemon.in

```diff
@@ -1,36 +1,62 @@
 #!/bin/sh
+### BEGIN INIT INFO
+# Provides:         ckb
```

Comment:
Should this not be `ckb-next`?

Reply:
Only matters for stuff dependent on the daemon, in the Required-Start and Required-Stop entries. `-next` can be added.